SU-HSDO/suhumsci
# Shs 54 percy local

## Summary
This adds the necessary items for Percy testing and updates the `README` with instructions on how to use.

## Steps to Test
1. Get access to the Percy organization from Joey if you haven't already.
2. Go to Percy.io and under 'Project Settings' copy the entire line under 'Project Token' into a `.env` file in the `humsci_basic` directory (same directory as the `package.json` for this theme). So the file will be `PERCY_TOKEN=XXX`
3. See the NOTES below for an additional step.
4. Install percy with `npm install`
5. Run `npm run visreg` and ensure it outputs links to the builds for both the Traditional/Colorful themes. 

## Notes
In order to have consistent testing, we need to ensure certain features are enabled/disabled before testing. These include:
* The Colorful site should use the new V2 Megamenu
* The Traditional site should use the old V1 Megamenu
* Both Colorful and Traditional sites should have the 'Use Animation Enhancements' feature turned off on their respective theme settings.

## Screenshots
![Screen Shot 2022-09-26 at 10 02 03 AM](https://user-images.githubusercontent.com/8405274/192297523-23a557e6-29c2-4d17-bccc-926edf3facb1.png)
